src/components/DownloadComponents/NextLessonContainer/NextLessonContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from "react";
+
+import Flex from "@/components/Flex";
+import Heading from "@/components/Typography/Heading";
+import NextLessonCard from "@/components/DownloadComponents/NextLessonCard/NextLessonCard";
+import { Span } from "@/components/Typography";
+import { NextLesson } from "@/node-lib/curriculum-api-2023/queries/lessonDownloads/lessonDownloads.schema";
+
+type NextLessonContainerProps = {
+  programmeSlug: string;
+  unitSlug: string;
+  nextLessons?: NextLesson[];
+  unitTitle?: string;
+};
+
+const NextLessonContainer: FC<NextLessonContainerProps> = ({
+  programmeSlug,
+  unitSlug,
+  nextLessons,
+  unitTitle,
+}) => {
+  return (
+    <Flex $flexDirection={"column"} $width={"100%"}>
+      <Heading tag={"h3"} $font={["heading-6", "heading-5"]} $mb={[32, 48]}>
+        More lessons in: <Span $color={"blue"}>{`${unitTitle}`}</Span>
+      </Heading>
+      {nextLessons && (
+        <Flex $flexDirection={["column", "row"]} $gap={[8, 12, 16]}>
+          {nextLessons.map((lesson: NextLesson, i: number) => {
```

Comment:
might be a bit cleaner to just `nextLessons.slice(0,2).map....` then you don't need the if and null return?

Reply:
actually just checked and the method used to get next lessons already limits to 3 (`getNextLessonsInUnit`)! so this may not be necessary at all